service/controller/drainer.go

```diff
@@ -159,6 +159,11 @@ func newDrainerResourceSets(config DrainerConfig) ([]*controller.ResourceSet, er
 		Region:          config.HostAWSConfig.Region,
 	}
 
+	awsHostClients, err := awsclient.NewClients(hostAWSConfig)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
```

Comment:
For the transition the drainer needs the EC2 client. This will be cleaned up later. TODO issue is tracked.

Reply:
That was the wrong client. We need the one from the controller context as it seems.